script.js

```diff
@@ -1,16 +1,37 @@
 // Write your JavaScript code here!
 
-window.addEventListener("load", function() {
+window.addEventListener("load", function () {
+    const form = document.querySelector("form");
+    form.addEventListener("submit", function (event) {
+        event.preventDefault();
+        let pilot = document.querySelector("input[name=pilotName]").value;
+        let copilot = document.querySelector("input[name=copilotName]").value;
+        let fuelLevel = document.querySelector("input[name=fuelLevel]").value;
+        let cargoLevel = document.querySelector("input[name=cargoMass]").value;
+        let list = document.getElementById("faultyItems");
+
+        formSubmission(document, list, pilot, copilot, fuelLevel, cargoLevel);
+    })
+
+
+    let listedPlanets;
+    // Set listedPlanetsResponse equal to the value returned by calling myFetch()
+    let listedPlanetsResponse = myFetch();
+    listedPlanetsResponse.then(function (result) {
+        listedPlanets = result;
+        console.log(listedPlanets);
+    }).then(function () {
+        console.log(listedPlanets);
+        // Below this comment call the appropriate helper functions to pick a planet fom the list of planets and add that information to your destination.
+        let planet = pickPlanet(listedPlanets)
+        let name = planet.name;
+        let diameter = planet.diameter;
+        let star = planet.star;
+        let distance = planet.distance;
+        let moons = planet.moons;
+        let imageUrl = planet.image;
+        addDestinationInfo(document, name, diameter, star, distance, moons, imageUrl);
+    })
```

Comment:
You want all this code to execute when the form is submitted (that is, when the button is clicked). So, this code needs to be inside the `form.addEventListener()...` function. I already mentioned above to change selecting the form to rather select the button.